src/main/webapp/js/common.js

```diff
@@ -870,3 +876,12 @@ function hideSingleCollapse(e) {
     $(heading).find('a.btn').hide();
 }
 
+/**
+ * updates the length of the textArea
+ * @param textAreaId - Id of text area for which char are to be counted
+ * @param charCountId - Id of Label to display length of text area
+ */
+function updateTextQuestionCharCount(textAreaId, charCountId) {
```

Comment:
This shouldn't belong in `common.js`, since only the submission pages should be concerned with this. This should go into `feedbackSubmissionEdit.js`